common/changes/office-ui-fabric-react/phkuo-msFabricColor_2017-08-15-18-14.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Fabric component: have the text color theme",
```

Comment:
This shows up in the public release notes...

Can you please be a lot more specific on what's changing?